teams/developer-relations/guide-to-being-a-manager/README.md

```diff
@@ -90,49 +90,110 @@ With regards to this checklist, reviewers should do the following:
 * Ensure that stage labels are being properly marked
 * When entire checklist is checked off for an Epic, then that Epic should be completed and linked within a pull request to `master` to indicate completion
 
-### General Checklist
+### General Checklist \(Applies to All Stages\)
 
-#### General Items \(Apply to All Stages\)
+Before going into a stage-by-stage breakdown, here's a general overview of things to keep in mind.
 
-* [ ] High-schooler friendly writing style, easy to understand
-* [ ] Correct numbering
-* [ ] Every card has code\* 
-* [ ] 1 scroll per card\*\*
-* [ ] Proper grammar, punctuation, capitalization, etc.
-* [ ] Acceptable Styling per the Pull Request Checklist
+#### Writing Style
 
-\*Exception being hard and medium cards in labs as well as activity cards that just introduce concepts
+First and foremost, we want all our developers to use a conversational writing style when making cards. This is because the target audience for our curriculum is high school students, and our goal is to make technical education as accessible as possible. Writing as if you're having a conversation will keep students more engaged. When in doubt, think **blog post** instead of **calculus textbook**.
 
-\*\* If you have to scroll more than once to view the whole card, the card is too long
+#### Numbering
+
+It's crucial that each card is numbered correctly so developers can keep lessons organized and students can follow a logical progression. BitProject uses two numbering schemes, one for Activities and one for Labs. 
+
+Here's an example of an Activity \(pretty self-explanatory\):
+
+![Numbering for Activity Cards](../../../.gitbook/assets/screen-shot-2020-03-29-at-3.00.05-pm.png)
+
+And here's an example of a Lab:
+
+![Numbering for Lab Cards](../../../.gitbook/assets/screen-shot-2020-03-29-at-3.06.54-pm.png)
+
+Each hard card is labelled as a single number.   
+Going off of the above example, **1.md is the first hard card**, and **2.md is the second hard card**. 
```

Comment:
if there is a mention of Hard/Medium/Easy here, then you should introduce that **first**